plugins/net_plugin/include/eosio/net_plugin/auto_bp_peering.hpp

```diff
@@ -402,7 +399,7 @@ class bp_connection_manager {
       if (invalid_message)
          return false;
```

Comment:
We could move these two lines right before line 391: `// peer key may have changed ... `